### PR TITLE
Link to npcap download page during Wireshark installation

### PR DESCRIPTION
**Pull request by gregmac**
image

### CODE REVIEW

File: bucket/wireshark.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "suggest": {
         "nmap": "nmap"
     },
+    "post_install": "Write-Host -fore yellow \"*********************`nNote: Wireshark needs a packet sniffing library to capture packets from your network interface, but this must be installed separately. You can download NPcap at https://nmap.org/npcap/#download`n*********************\";",
```
Comment (Ash258) [inline bucket/wireshark.json]: Or just add npcap manifest and depends on it.
```suggestion
    "notes": [
        "Wireshark needs a packet sniffing library to capture packets from your network interface, but this must be installed separately",
        "You can download NPcap at https://nmap.org/npcap/#download",
    ],
```

**Comment by linsui** [comment]
nmap has been suggested which contains npcap.

**Comment by gregmac** [comment]
I didn't actually realize nmap had npcap (the manifest for nmap doesn't mention it), so in addition to taking @Ash258's suggestion to use `notes` I explicitly call that out. 

Should we update the nmap manifest to mention it includes npcap? This would make it searchable in scoop, at the least. Right now the only way to figure that out is to visit the [nmap download page](https://nmap.org/download.html) and read a few paragraphs down -- not obvious if you're just trying to use wireshark.

File: bucket/wireshark.json [hunk 2]
```diff
@@ -31,6 +31,11 @@
     "suggest": {
         "nmap": "nmap"
     },
+    "notes": [
+        "Wireshark needs a packet sniffing library to capture packets from your network interface.",
+        "Please either install the 'nmap' package or install NPcap from https://nmap.org/npcap/#download."
+    ],
+    "post_install": "Write-Host -fore yellow \"*********************`nNote: Wireshark needs a packet sniffing library to capture packets from your network interface, but this must be installed separately. You can download NPcap at https://nmap.org/npcap/#download`n*********************\";",
```
Comment (linsui) [inline bucket/wireshark.json]:
```suggestion
```

File: bucket/wireshark.json [hunk 3]
```diff
@@ -31,6 +31,11 @@
     "suggest": {
         "nmap": "nmap"
```
Comment (linsui) [inline bucket/wireshark.json]:
```suggestion
         "pcap": "nmap"
```

Maybe you can also change the suggest as we only need npcap in nmap. This is more clear.

**Comment by gregmac** [comment]
Already fixed in #2930

**Pull request closed without merging**